tests/modules/summary_report/main.nf.test

```diff
@@ -0,0 +1,44 @@
+nextflow_process {
+
+    name "Test Process SUMMARY_REPORT"
+    script "modules/local/summary_report.nf"
+    process "SUMMARY_REPORT"
+
+    test("Should run without failures") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ],
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/summary_report/test.vif.evidence.bam', checkIfExists: true),
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/summary_report/test.vif.evidence.bam.bai', checkIfExists: true),
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/viralintegration/testdata/summary_report/test.vif.init.filtered.abridged.tsv', checkIfExists: true),
```

Comment:
These need to updated to the fixed results

Reply:
https://github.com/nf-core/viralintegration/issues/37